src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private OperationType operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(OperationType operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public OperationType getOperation() {
```

Comment:
Follow the encapsulation principle. If the getters and setters are not used outside of the model package, consider reducing their visibility to package-private.